ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/trace/specs/trace-transaction/trace_transaction_13_1_trace.json

```diff
@@ -0,0 +1,88 @@
+{
+  "comment" : "call the prior contract then call log contract 00a0..00.",
+  "request" : {
+    "jsonrpc" : "2.0",
+    "method" : "trace_transaction",
+    "params" : [ "0x6b9b967cfbeedeb7f0f4956b8103075ddfcea26c01d6d5dc3f9e2ed2ec9c42c0" ],
+    "id" : 159
+  },
+  "response" : {
+    "jsonrpc" : "2.0",
+    "result" : [ {
+      "action" : {
+        "from" : "0x627306090abab3a6e1400e9345bc60c78a8bef57",
+        "gas" : "0xff2e26",
+        "init" : "0x60006000600060006000738f0483125fcb9aaaefa9209d8e9d7b9c8b9fb90f5af1600060006000600060007300a00000000000000000000000000000000000005af1",
+        "value" : "0x0"
+      },
+      "blockHash" : "0x425ea2cf9bb89c09f131ab531958c4d189b5d047fd924d74f25d49cf47a5a8f1",
+      "blockNumber" : 19,
+      "result" : {
+        "address" : "0x9fbda871d559710256a2502a2517b794b482db40",
+        "code" : "0x",
+        "gasUsed" : "0x1c39"
+      },
+      "subtraces" : 2,
+      "traceAddress" : [ ],
+      "transactionHash" : "0x6b9b967cfbeedeb7f0f4956b8103075ddfcea26c01d6d5dc3f9e2ed2ec9c42c0",
+      "transactionPosition" : 1,
+      "type" : "create"
+    }, {
+      "action" : {
+        "callType" : "call",
+        "from" : "0x9fbda871d559710256a2502a2517b794b482db40",
+        "gas" : "0xfb2ea9",
+        "input" : "0x",
+        "to" : "0x8f0483125fcb9aaaefa9209d8e9d7b9c8b9fb90f",
+        "value" : "0x0"
+      },
+      "blockHash" : "0x425ea2cf9bb89c09f131ab531958c4d189b5d047fd924d74f25d49cf47a5a8f1",
+      "blockNumber" : 19,
+      "result" : {
+        "gasUsed" : "0x138e",
+        "output" : "0x"
+      },
+      "subtraces" : 1,
+      "traceAddress" : [ 0 ],
+      "transactionHash" : "0x6b9b967cfbeedeb7f0f4956b8103075ddfcea26c01d6d5dc3f9e2ed2ec9c42c0",
+      "transactionPosition" : 1,
+      "type" : "call"
+    }, {
+      "action" : {
+        "address" : "0x8f0483125fcb9aaaefa9209d8e9d7b9c8b9fb90f",
+        "balance" : "0x0",
+        "refundAddress" : "0x0000000000000000000000000000000000000000"
+      },
+      "blockHash" : "0x425ea2cf9bb89c09f131ab531958c4d189b5d047fd924d74f25d49cf47a5a8f1",
+      "blockNumber" : 19,
+      "result" : null,
+      "subtraces" : 0,
+      "traceAddress" : [ 0, 0 ],
```

Comment:
this matches https://github.com/hyperledger/besu/blob/365a7f95258192774d59154160d862b7887813c4/ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/trace/specs/trace-get/trace_get_13_1_2_trace.json